docs/app/Examples/elements/Step/Content/StepLinkExamples.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Step } from 'stardust'
 
 // eslint-disable-next-line no-alert
-const handleClick = () => alert('Clicked!')
+const handleClick = (e) => {alert('Clicked!'), console.log(e.target)}
```

Comment:
Would prefer we avoid `alert()` for examples. We've been using stateful examples when handling interactivity like this.  We update the example with some kind of output to show what is going on.  I think it is a cleaner representation, but also more complete in that it shows users how to fully integrate the component with their own view.  Here are some examples:


### Radio Groups
http://technologyadvice.github.io/stardust/addons/radio#radio-group

### Table Selectable 
http://technologyadvice.github.io/stardust/collections/table#selectable

### Accordion activeIndex
http://technologyadvice.github.io/stardust/modules/accordion#active-index

### Dropdown
http://technologyadvice.github.io/stardust/modules/dropdown#async-options